script.motr/resources/skins/Default/720p/convertdialog.xml

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you rename the windows to script-motr-xxxx. Reason is to make sure there is no clash with skin dialogs